src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/PropertyPageResources.resx

```diff
@@ -252,4 +252,16 @@
   <data name="chkSqlCodeDebuggingText" xml:space="preserve">
     <value>Enable SQL Server debugging</value>
   </data>
+  <data name="chkRemoteDebugEnabledHelpText" xml:space="preserve">
+    <value>Enable remote debugging</value>
+  </data>
+  <data name="RemoteDebug" xml:space="preserve">
+    <value>Use remote m_achine</value>
```

Comment:
Is the _ intended or is it a typo?

Reply:
Intended, it indicates a _mnemonic_ character in WPF. In Windows Forms, & is the same indicator.

Reply:
Thanks for the clarification 👍

Reply:
I copied the mnemonic from legacy. At the same time I noticed we don't have mnemonics for anything else on this property page. Filed #5797 to track that.